junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ExtensionContext.java

```diff
@@ -262,20 +262,25 @@ default Store getStore() {
 	Store getStore(Namespace namespace);
 
 	/**
-	 * {@code Store} provides methods for extensions to save and retrieve data.
+	 * Provides methods for extensions to save and retrieve data.
+	 *
+	 * <p>For better isolation and type-safety, consider accessing the
+	 * {@code Store} via a {@link StoredValues} instance.
+	 *
+	 * @see StoredValues
 	 */
 	interface Store {
 
 		/**
-		 * Get the value that is stored under the supplied {@code key}.
+		 * Returns the value that is stored under the supplied {@code key}.
```

Comment:
We always use "Get" instead if "Gets" etc. because we want to describe the action carried out by the method not the method itself.